modules/concom/vue/views/StaffStructurePage.js

```diff
@@ -61,12 +63,17 @@ function setup() {
   return {
     fetchUser,
     fetchDivisions,
+    prepareSidebar,
+    changeSidebar,
+    closeSidebar,
+    saveSidebarData,
     divisions,
     canEditRbac,
     showSidebar,
     sidebarName,
     sidebarData,
-    sidebarDisplayClass
+    sidebarDisplayClass,
+    sidebarTempData: []
```

Comment:
I will remove `sidebarTempData` in a future commit, I originally had a plan for this before moving logic to composables.